app/components/Footer.jsx

```diff
@@ -67,13 +97,37 @@ export default class Footer extends Component {
           <input
             type="radio"
             id="locale-frFR"
-            onClick={(item) => this.props.toggleLocale(item)}
+            onChange={(item) => this.props.toggleLocale(item)}
             name="language"
             value="fr-FR"
             checked={this.isLocaleChecked('fr-FR')}
           />
           <label htmlFor="locale-frFR">FR</label>
         </span>
+        <span className="viewLink">
+          <button
+            className="showThemesBtn"
+            onClick={this.toggleIsThemesBarOpen}
+          >Themes</button>
+        </span>
+        <div className={`themes ${this.state.isThemesBarOpen ? 'open' : ''}`} >
+          <h3>Themes</h3>
+          <ul>
+            {this.themes.map((theme, index) => {
+              return (
+                <li key={index} className="theme-item">
```

Comment:
Using a the theme's name as the key would make more sense, because the order of the themes in the array is not significant, as far as I can see.